conans/client/rest/conan_requester.py

```diff
@@ -13,6 +14,12 @@ class ConanRequester(object):
 
     def __init__(self, config, http_requester=None):
         self._http_requester = http_requester if http_requester else requests.Session()
```

Comment:
I think it would be more clear:

```
if http_requester:
    self._http_requester = http_requester
else:
    self._http_requester = requests.Session()
    adapter = HTTPAdapter(max_retries=config.retry)
    self._http_requester.mount("http://", adapter)
    self._http_requester.mount("https://", adapter)
```

Reply:
done